Developer-Guide--Set-Up-With-Docker.md

```diff
@@ -179,3 +179,31 @@ If the `rails db:autotest_run` fails, you can still run the tests manually in yo
 5. Go to the Test Results tab and click on "Run Tests".
 6. Wait a minute, and then refresh the page. Go back to the Test Results tab. You should see that two tests have been run, and that both have passed.
 7. Repeat for the other assignments that you want to run tests for.
+
+### Setting up action mailer
+
+If you plan on doing work that involves sending/recieving emails from MarkUs, you will need to configure action mailer. To get you started quickly, here we will talk specifically about configuring action mailer with a gmail account using the `:smtp` delivery method.
+
+1. Find the `config/settings/development.yml` file.
```

Comment:
Okay so we don't want students to modify this file, since it's tracked by git and they might accidentally commit a gmail username and password.

Instead, modify the instructions to have developers create a new file `config/settings.local.yml`, and have them add the configuration there. (Please also test this yourself locally---the [config gem documentation](https://github.com/rubyconfig/config) indicates this is a default, but I haven't tried it myself.

*Then* assuming it works, also submit a PR on the MarkUs repo to add `config/settings.local.yml` to the `.gitignore` file so that it won't be tracked by git.

Reply:
I have tested this. It works and MarkUs is already configured to automatically not track the `config/settings.local.yml` by git (and thus an additional PR is not needed).